### PR TITLE
Use freckle api v2

### PR DESCRIPTION
**Pull request by mauro-oto**
Hey @etagwerker, @schmierkov, 

This PR fixes https://github.com/ombulabs/pecas/issues/19. By moving from https://github.com/ryanlecompte/letsfreckle-client (no activity for ~2 years) to https://github.com/timcraft/freckles, we will effectively switch from using Freckle API v1 to v2. A new Freckle token will have to be generated for using v2, as the current one in your `.env` will only work for v1.

We're currently using a forked version of the `freckles` gem though, because it didn't support a method I needed. I have opened a PR for them here: https://github.com/timcraft/freckles/pull/1

Please check it out, thanks!

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.1.2@freckler
+2.1.2@pecas
```
Comment (etagwerker) [inline .ruby-version]: @mauro-oto Could you remove `.ruby-version` from the repo and add it to `.gitignore`?

File: Gemfile [hunk 2]
```diff
@@ -13,8 +13,9 @@ gem 'coffee-rails', '~> 4.0.0'
 # See https://github.com/sstephenson/execjs#readme for more supported runtimes
 # gem 'therubyracer',  platforms: :ruby
 
-gem 'dotenv'
-gem 'letsfreckle-client', branch: :master, require: 'letsfreckle', github: 'ryanlecompte/letsfreckle-client'
+gem 'dotenv-rails'
+gem 'freckles', github: 'ombulabs/freckles',
+                branch: 'add-last-page-to-responses-with-rel-last-link'
```
Comment (etagwerker) [inline Gemfile]: @mauro-oto Could we use this project now? https://github.com/timcraft/freckles

I think that your PR got implemented in `master`

File: lib/tasks/importer.rake [hunk 3]
```diff
@@ -9,31 +28,31 @@ namespace :import do
...
+    save_entries_for(result)
+
+    if result.try(:last_page)
+      last_page = result.last_page.match(/page=(\d+)/)[1].to_i
+      [2..last_page].each do |page|
+        result = freckle_client.get_entries(from: start_date, to: end_date,
+                                            page: page)
+        save_entries_for(result)
+      end
     end
```
Comment (etagwerker) [inline lib/tasks/importer.rake]: @mauro-oto Could you move this code and `#save_entries_for` to a service class? It'll make it easier to test its behavior.

**Comment by etagwerker** [comment]
@mauro-oto Just finished reviewing your changes. Please take a look!

**Comment by mauro-oto** [comment]
@etagwerker done! I added some basic coverage for `FreckleService`, let me know if you think we need any more tests.

**Comment by etagwerker** [comment]
@mauro-oto Looks good, thanks! :+1:

**Pull request merged**